src/GovUk.Education.ExploreEducationStatistics.Notifier/Functions/ReleaseNotifier.cs

```diff
@@ -0,0 +1,204 @@
+        {
+            var releaseSupersededPubSubsQuery = new TableQuery<SubscriptionEntity>()
+                .Where(TableQuery.GenerateFilterCondition("PartitionKey", QueryComparisons.Equal,
+                    supersededPublication.Id.ToString()));
+            var supersededPublicationSubscriberEmails = await GetSubscriberEmails(
+                subscribersTable, releaseSupersededPubSubsQuery);
+
+            var numSupersededSubscriberEmailsSent = 0;
+
+            foreach (var email in supersededPublicationSubscriberEmails)
+            {
+                if (sentEmails.Contains(email))
+                {
+                    continue;
+                }
+
+                SendSupersededSubscriberEmail(email, supersededPublication,
+                    msg, notificationClient, config);
+                sentEmails.Add(email);
+                numSupersededSubscriberEmailsSent++;
+            }
+            logger.LogInformation("Emailed {NumSupersededPublicationEmailsSent} subscribers from a superseded publication",
+                numSupersededSubscriberEmailsSent);
+        }
+        logger.LogInformation("Sent {TotalNumEmailsSent} emails in total to subscribers",
+            sentEmails.Count);
+    }
+
+    private static async Task<List<string>> GetSubscriberEmails(
+        CloudTable table,
+        TableQuery<SubscriptionEntity> query)
+    {
+        var emails = new List<string>();
+
+        TableContinuationToken? token = null;
+        do
+        {
+            var resultSegment =
+                await table.ExecuteQuerySegmentedAsync(query, token);
+            token = resultSegment.ContinuationToken;
+
+            var newEmails = resultSegment.Results
+                .Select(entity => entity.RowKey)
+                .ToList();
+            emails.AddRange(newEmails);
+        } while (token != null);
+
+        return emails;
+    }
+
+    private void SendSubscriberEmail(
+        string email,
+        ReleaseNotificationMessage msg,
+        NotificationClient notificationClient,
+        IConfigurationRoot config)
+    {
+        var baseUrl = config.GetValue<string>(BaseUrlName);
+        var webApplicationBaseUrl = config.GetValue<string>(WebApplicationBaseUrlName).AppendTrailingSlash();
+        var tokenSecretKey = config.GetValue<string>(TokenSecretKeyName);
+
+        var unsubscribeToken =
+            _tokenService.GenerateToken(tokenSecretKey, email, DateTime.UtcNow.AddYears(1));
+
+        var values = new Dictionary<string, dynamic>
+        {
+            { "publication_name", msg.PublicationName },
+            { "release_name", msg.ReleaseName },
+            {
+                "release_link",
+                $"{webApplicationBaseUrl}find-statistics/{msg.PublicationSlug}/{msg.ReleaseSlug}"
+            },
+            {
+                // NOTE: update_note not used by original release email template - only with amendments
+                "update_note", msg.UpdateNote
+            },
```

Comment:
Would be a quick improvement to only add this to the dictionary if the amendment flag is true, removing the need for the comment.

There's one other case of this as well.